src/Sulu/Component/CustomUrl/Manager/CustomUrlManager.php

```diff
@@ -68,14 +73,20 @@ class CustomUrlManager implements CustomUrlManagerInterface
      */
     private $environment;
 
+    /**
+     * @var DocumentDomainEventCollectorInterface
+     */
+    private $documentDomainEventCollector;
+
     public function __construct(
         DocumentManagerInterface $documentManager,
         DocumentInspector $documentInspector,
         CustomUrlRepository $customUrlRepository,
         MetadataFactoryInterface $metadataFactory,
         PathBuilder $pathBuilder,
         WebspaceManagerInterface $webspaceManager,
-        $environment
+        $environment,
+        DocumentDomainEventCollectorInterface $documentDomainEventCollector
```

Comment:
breaking change which we maybe need to document in the UPGRADE.md

Reply:
👍